include/odp/api/spec/ml_types.h

```diff
@@ -329,6 +329,15 @@ typedef struct odp_ml_capability_t {
 
 /** Machine learning configuration parameters */
 typedef struct odp_ml_config_t {
+	/**
+	 * Engine ID to be configured.
+	 *
+	 * In a system with multiple ML engines, this parameter selects the engine
+	 * to be configured. The default value is 0. Engine ID should be in the range
+	 * 0..num_engines, where num_engines can be fetched using odp_ml_num_engines().
```

Comment:
The range should be `0..num_engines - 1`, right (like documented in `odp_ml_capability()`)?

Reply:
Typo still here?